applications/ws-affiliation-rnsr/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM inistcnrs/lodex-workers-python:4.0.11
+RUN pip install dvc[all]==2.48 fasttext==0.9.2
+WORKDIR /app/public
+COPY /models /app/public/
+RUN npm install -g node-gyp && \
```

Comment:
We stopped here.
But the next step would be to invoke a `dvc pull`.
And at some point, we should configure the dvc remote (login/password taken from the config.json).
[lodex-makefile](https://github.com/Inist-CNRS/ezmaster-apps/blob/b1389ea9b224daba89e1b2afc84d3d832e3fa3c6/applications/lodex-makefile/docker-entrypoint.sh#L18) can be inspiring:

```sh
WEBDAV_LOGIN=`node -e 'process.stdout.write(require("./config.json").webdav?.login??"")'`
WEBDAV_PASSWORD=`node -e 'process.stdout.write(require("./config.json").webdav?.password??"")'`
```

Reply:
The previous comment was a false good idea: it's not in the `docker-entrypoint.sh` that we should get the webdav configuration (because it is run at every start of the image).

A better practice is to use environment variable **at build**.
So we need to modify the `build` script in `package.json`, and document that it is required to set `WEBDAV_URL`, `WEBDAV_LOGIN` and `WEBDAV_PASSWORD` in the environment where we launch `npm run build` to build the Docker image.

Reply:
It's a bit more complicated, if we want to keep login and password secret.
See [Pass secure information for building Docker images](https://medium.com/marionete/pass-secure-information-for-building-docker-images-8adeafe08355).

Oh. And we should install DVC only in build stage, not in the final image.